code-challenges.test.js

```diff
@@ -22,7 +24,27 @@ const fibonacciLength2 = 10
 
 // b) Create the function that makes the test pass.
 
+// describe("fibMaker", () => {
+//   it("returns array containing Fibonacci sequence with the length of the array determined by input number", () => {
+//     expect(fibMaker(6)).toEqual([1, 1, 2, 3, 5, 8])
+//     expect(fibMaker(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+//   })
+// })
+
 // Pseudo code:
+// function name: fibMaker
+// input: number over 2
+// output: array of Fibonacci sequence equal in length to the number entered
+// process: for loop, .slice() built-in method
+
+// const fibArr = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144]
+
+// const fibMaker = (number) => {
+//   for (i = 0; i < fibArr.length; i++)
+//   return fibArr.slice(0, number)
```

Comment:
Interesting approach, though what if the input number was 500? Remember you always want to write code that is dynamic, meaning it's not limited to just the given test variables.